docs/views/views.mdx

```diff
@@ -65,8 +69,16 @@ Each of the following views can be passed as the `view` option when creating a n
   <tr>
     <td>space-separated-values</td>
     <td>
-      Table view of inputs, outputs, and labels which are space-separated words. Useful
-      for tasks such as part-of-speech tagging.
+      Table view of inputs, outputs, and labels which are space-separated words.
+      Useful for tasks such as part-of-speech tagging.
+    </td>
+  </tr>
+  <tr>
+    <td>rag</td>
+    <td>
+      Show the input to a generator that can access information from other
```

Comment:
In first sentence maybe say this is "retreival-augmented generation" and then add a sentence or two describing the data format.